.travis.yml

```diff
@@ -50,6 +50,10 @@ before_script:
   # Remember the Drupal installation path.
   - DRUPAL_DIR=$(pwd)/drupal
 
+  # Download Drupal console so we can run the test for it. Skip this for the
+  # coding standards test.
+  - test ${TEST_SUITE} == "PHP_CodeSniffer" || composer require symfony/dom-crawler:2.8.11 drupal/console-core:1.0.0-rc5 drupal/console:1.0.0-rc5 --working-dir=$DRUPAL_DIR
```

Comment:
It's better if you don't hardcode the versions, otherwise we will have to maintain this manually and check every day if there is a new version available of Drupal Console.

If you need 1.0.0-rc5 as a minimum version, can you check if you can get it to work with this command?

```
$ composer require drupal/console:~1.0.0-rc5
```

Also we don't depend on `symfony/dom-crawler` and `drupal/console-core`, only on `drupal/console`. If you need to add these to work around a dependency problem inside Drupal Console then you need to document this really clearly. The documentation says:

```
# Download Drupal console so we can run the test for it.
```

It doesn't say anything about why we need `dom-crawler`. I think if we maybe update the documentation like this it would be clearer to future maintainers why we are doing weird stuff:

```
# Download Drupal console so we can run the test for it. Skip this for the
# coding standards test.
# Drupal console requires a newer version of symfony/dom-crawler than
# Drupal core. Work around this by updating core to this version.
```

Also don't hardcode `dom-crawler` to 2.8.11, that will just make it necessary to look at this sooner, what if in the future Drupal 8.2.x is released which requires 2.8.12?

I think we should probably report this to Drupal Console and make sure they are compatible with the same version of `symfony/dom-crawler` that Drupal core uses.